src/__tests__/operate.test.jsx

```diff
@@ -0,0 +1,23 @@
+import operate from '../Logic/operate';
+
```

Comment:
- [ ] Kindly use jest snapshots to test all components of your application as demanded by the project requirements.
- [ ] Kindly use React Test Library to simulate user interactions as demanded by the project requirements.